editor/js/libs/app.js

```diff
@@ -27,8 +33,13 @@ var APP = {
 
 			var project = json.project;
 
+			vrEnabled = () => project.vr;
```

Comment:
Can you please explain why `vrEnabled` is a function? Besides, couldn't you just test for `renderer.xr.enabled` in `play()` and `stop()`?

Reply:
This is to pick up the project configuration flag. `renderer.xr.enabled` depends on the browser, and is independent of the project configuration. I added a function here because it provided a convenient handle on the project.vr property - in published zips this doesn't change, but in the editor it is possible to turn the property on and off between calls to `play()` and `stop()`, so we need a live handle to the `project.vr` field. An alternative would be to keep a reference to the `project` and read it from there.

Reply:
> renderer.xr.enabled depends on the browser, and is independent of the project configuration. 

As far as I understand the flag is only set to `true` if the project has enabled `vr` (see https://github.com/mrdoob/three.js/pull/18630/files#diff-9a03039070ec9199ce533fdc1f2c9fc1R39-R42). Hence, you should be able to directly use `renderer.xr.enabled`.

Reply:
I understand what the issue is.

The problem is that we're not setting `renderer.xr.enabled` when `project.vr` is `false` (happens with `project.shadows` too).

I'll fix it.

Reply:
https://github.com/mrdoob/three.js/commit/814b447262454ae36c7190555d09e57454947a19